### DISCUSSION TITLE
feature:  getting side by side terminal preview for web development , dont wanna switch tabs from neovim and then go to chrome to check ... can you add browser  like vscode  and overall it is good

### DISCUSSION THREAD

**Discussion by jscodinity** [LazyVim/LazyVim#2662] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

no its is not a problem but  when i make a web projects and run the projects and i switch terminal to chrome see what i  make and what changes should i do and then back to terminal and doing this again and again its like switching from one tab to another tab that why i recommended

### Describe the solution you'd like

when i run project i want the lazyvim terminal open browser in terminal by split the terminal

### Describe alternatives you've considered

nothing more

**Comment by dpetka2001** [comment]
The browser can't be opened in Neovim's terminal. It's not the same as in VsCode, which is a GUI application and can open the browser embedded in a split VsCode window.

**Comment by kobzar** [comment]
Maybe you nedd something like this
https://github.com/ray-x/web-tools.nvim

**Comment by jscodinity** [comment]
Can we do ios development in lazyvim
